### PR TITLE
Enable checkstyle and spotbugs tasks

### PR DESCRIPTION
**Pull request by aashikam**
## Purpose
$title

Fixes: https://github.com/ballerina-platform/ballerina-standard-library/issues/230

### CODE REVIEW

File: rabbitmq-native/build.gradle [hunk 1]
```diff
@@ -17,17 +17,52 @@
...
+
+spotbugsMain {
+    effort "max"
+    reportLevel "low"
+    reportsDir = file("$project.buildDir/reports/spotbugs")
+    reports {
+        html.enabled true
+        text.enabled = true
+    }
+    def excludeFile = file("${rootDir}/spotbugs-exclude.xml")
```
Comment (chamil321) [inline rabbitmq-native/build.gradle:56]:
```suggestion
    def excludeFile = file("spotbugs-exclude.xml")
```

Comment (aashikam) [reply rabbitmq-native/build.gradle:56]: This failed the check. Worked with `file("${rootDir}/spotbugs-exclude.xml")`

File: build.gradle [hunk 2]
```diff
@@ -28,6 +28,7 @@ ext.puppycrawlCheckstyleVersion = "8.18"
 ext.slf4jVersion = "1.7.30"
 ext.amqpVersion = "5.7.3"
 ext.stdlibCryptoVersion = project.stdlibCryptoVersion
+ext.puppycrawlCheckstyleVersion = project.puppycrawlCheckstyleVersion
```
Comment (chamil321) [inline build.gradle:31]: Remove L27

File: rabbitmq-native/build.gradle (hunk 1 above)
Comment (chamil321) [reply rabbitmq-native/build.gradle:56]: Ack

**Review by chamil321 — APPROVED** [review]

**Pull request merged**